src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogConfiguration.cs

```diff
@@ -40,14 +40,20 @@ public class DatadogConfiguration
         /// </summary>
         public bool UseSSL { get; set; }
 
-        public DatadogConfiguration() : this(DDUrl, DDPort, true) {
+        /// <summary>
+        /// Use TCP or HTTP.
+        /// </summary>
+        public bool UseHTTP { get; set; }
```

Comment:
we changed this to `useTCP` on the lambda as we want to enable HTTP by default.

Reply:
HTTP is indeed much more appropriate for Agentless logging. Let's have this as the default behaviour and allow to use TCP for users that might have a high volume of logs and that require as close to real time as possible.